src/com/amazon/ion/benchmark/Main.java

```diff
@@ -251,6 +251,8 @@ public class Main {
         + "  -Q --input-ion-schema <file_path>      This option will specify the path of Ion Schema file which contains all constraints that the "
             + "generated Ion data would conform with.\n"
 
+        + "  -M --seed <seed_value>   This option will be specified when users would like to get the same random data from the same schema file.\n"
```

Comment:
We should be clear about what type of value is expected. An integer? What size?

Reply:
> We should be clear about what type of value is expected. An integer? What size?

These information will be clarified in the following commit.